Structure_Engine/Query/ElasticValues.cs

```diff
@@ -33,12 +33,12 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Gets the spring values from a constraint as a double array. Values returned in the following order: TransX, TranxY, TransZ, RotX, RotY, RotZ.")]
-        [Input("constraint", "The constraint to spring values from.")]
+        [Description("Gets the spring values from a Constraint6DOF as a double array. Values returned in the following order: TransX, TranxY, TransZ, RotX, RotY, RotZ.")]
+        [Input("constraint", "The Constraint6DOF to spring values from.")]
         [Output("springValues", "The elastic values, or spring values, from a constraint as a double array in the following order: TransX, TransY, TransZ, RotX, RotY, RotZ.")]
         public static double[] ElasticValues(this Constraint6DOF constraint)
         {
-            return new double[] 
+            return constraint.IsNull() ? null : new double[]
```

Comment:
scope to return a `new double[6]` here instead of null. Kind of similar discussion to the one for setting orientations and area for that matter. Does a null constraint means it has invalid elastic values (null) or does it mean it has zero stiffness (`new double[6]`).

Not 100% sure myself again, but returning null here I see as equivalent to returning double.NaN for area, while returning `new double[6]` I would see as equivalent to returning 0 area.

Might not be 100% equivalent, but pretty close.

Glad to get others opinions on this. @al-fisher @rolyhudson @peterjamesnugent @JosefTaylor @FraserGreenroyd